packages/waku/src/lib/builder.ts

```diff
@@ -61,6 +61,9 @@ const analyzeEntries = async (entriesFile: string) => {
       ),
     ],
     ssr: {
+      resolve: {
+        conditions: ["react-server"],
+      },
```

Comment:
Do you think we need to keep L69-71??

Reply:
I think some non-ssr modules might include some `react-server '-only libraries. I think the bugs we met only happened in the SSR/Server module. if we remove it, I think the client-only code might error now.  (that's how I imagine the logic from Vite)

Not sure, let me try in the production

Reply:
Let me add some real-prototype

Reply:
Yep, we can

Reply:
You can try this demo

https://github.com/himself65/waku/tree/11840a4950d020fefdd53781103f09f5c6a0ca7c/playground/react-server

Not sure if we need add this to the e2e test

Reply:
I think we need to add some ecosystem-ci to ensure everything ok in the future